File: src/args.go
/*
#	What Is This: programming samples
#	Author: Makoto Takeshita <takeshita.sample@gmail.com>
#	URL: http://simplesandsamples.com
#	Version: UNBORN
#	
#	Usage:
#	 1. git clone https://github.com/takeshitamakoto/sss.git
#	 2. change the directory name to easy-to-use name. (e.g. sss -> sample)
#	 3. open sss/src/filename when you need any help . 
#	
*/

package main
import (
	"os"
	"fmt"
)
func main() {
	for i:=0; i<len(os.Args); i++ {
		fmt.Printf("%s\n", os.Args[i])
	}
}

File: src/args.vb
'
'	What Is This: programming samples
'	Author: Makoto Takeshita <takeshita.sample@gmail.com>
'	URL: http://simplesandsamples.com
'	Version: UNBORN
'	
'	Usage:
'	 1. git clone https://github.com/takeshitamakoto/sss.git
'	 2. change the directory name to easy-to-use name. (e.g. sss -> sample)
'	 3. open sss/src/filename when you need any help.
'	

Imports System	'for System.Console.WriteLine

Class Test
	Shared Sub Main()
		Dim args As String() = Environment.GetCommandLineArgs()
		For Each m As String In args
			Console.WriteLine("{0}",m)
		Next
	End Sub
End Class

